lib/bootloader_setup.pm

```diff
@@ -207,7 +207,7 @@ sub boot_local_disk {
         }
         if (match_has_tag 'inst-slof') {
             diag 'specifying local disk for boot from slof';
-            type_string_very_slow "boot /pci\t/sc\t5";
+            type_string_very_slow "boot /pci\t/sc\t6";
```

Comment:
In fact, I don't know how to get the scsi device id since it is depend on slof. I get the current scsi device id just by viewing the output of serial of slof. Although this value will changed if new device added, but such situation should be very rare. So I suggest to merge it first to escape blocking all migration test on ppc64le, then deep study how to calculate the device id and submit a new PR to fix it. WDYT?

Reply:
I wonder why did it change so fast I changed it less than 4 months ago apparently... ... @lemon-suse can you double check the workers? or at least run the same job on all of the ppc64 workers?... But I think we can do that as part of a separate PR

Reply:
@lemon-suse, without `VIRTIO_CONSOLE` booting is fine, so this can be a verification run: https://openqa.suse.de/tests/2297446#step/boot_to_desktop/6 :)

Reply:
@okurz I understand your meaning now, updated the code following your suggestion.